src/navigator/IFrameNavigator.ts

```diff
@@ -424,6 +424,11 @@ export default class IFrameNavigator implements Navigator {
     this.headerMenu = headerMenu;
     this.mainElement = mainElement;
     try {
+      var wrapper = HTMLUtilities.findRequiredElement(
+        mainElement,
+        "main#iframe-wrapper"
+      ) as HTMLElement;
```

Comment:
The `findRequiredElement` function now returns an `HTMLElement`, so the `as` typecasting is no longer necessary

Reply:
ok, i missed that, and i'll be refactoring here a bit more too, just needed to make sure the majority of the functionality still works. e.g. next / previous page